aerial_transportation_plugin_name: aerial_transportation/hydrus

nav_vel_limit: 0.3
vel_nav_threshold: 0.2

approach_threshold: 0.1
approach_count: 1.0
object_head_direction: true

object_height: 0.05 # set to the grasp height
grasping_height_offset: 0.0

ascending_speed: 0.1
transportation_count: 1.0

torque_min_threshold: 0.2
torque_max_threshold: 0.5

j1:
        approach_angle: 0.8
        hold_angle: 1.29
        tighten_angle: 0.08 # + 0.08rad(5deg)

j2:
        approach_angle: 1.15
        hold_angle: 1.51
        tighten_angle: 0.08 # + 0.08rad(5deg)

j3:
        approach_angle: 0.8
        hold_angle: 1.34
        tighten_angle: 0.08 # + 0.08rad(5deg)